docker/pypi/wmagent/init.sh

```diff
@@ -313,7 +315,10 @@ check_databases() {
     esac
 
     # Checking CouchDB:
-    _check_couch
+    [[ -n $COUCH_HOST ]] && [[ -n $COUCH_PORT ]] && [[ -n $COUCH_USER ]] && [[ -n $COUCH_PASS ]] &&
```

Comment:
@mapellidario , add a default value of:
 ```
 couchdbCred=false
```
at the line just above this one. Otherwise it will break the execution with a syntax error on line 321, in the cases when even one of the CouchDb credential variables is missing  at `WMAgent.secrets`

Reply:
I implemented this comment in another PR :)